src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -1,12 +1,99 @@
 @import '~style/variables';
 
+.categoryNav {
+  background-color: $color-white;
+  display: flex;
+  height: 48px;
+  align-items: center;
+  padding-left: 20px;
+  width: 100%;
+  white-space: nowrap;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+
+  @include for-desktop-up {
+    height: 80px;
+    padding-left: 30px;
+  }
+}
+
 .link {
   display: inline-block;
-  padding: 10px;
+  padding: 0 20px;
   text-decoration: none;
+  height: 23px;
```

Comment:
It's hard to vertically centre text when setting the height.

![image](https://user-images.githubusercontent.com/4443482/31105646-7c455e2c-a831-11e7-830e-f468ac4e1416.png)

By setting only padding you'll get better results, e.g. `padding: 9px 20px 12px;` (or whatever works) and remove the height property.

Also, for good UX links should have a minimum tap target of at least 40px (less than that will fail some accessibility tests). Nothing serious but I'd mention that to Nick.